packages/serverpod_client/lib/src/client_method_stream_manager.dart

```diff
@@ -495,9 +495,9 @@ final class ClientMethodStreamManager {
       await webSocket.sink.close();
     } finally {
       /// Close any still open streams with an exception.
```

Comment:
Should the comment be moved as well? Or remove it if it's clear enough by the code?

Reply:
Good suggestion! Moved it 👍